### PR TITLE
Loadout update

### PR DESCRIPTION
**Pull request by HeroesMaker**
simple update on loadouts and also updated weapon-list.sqf

### CODE REVIEW

File: loadouts/content/broma-weapon-list.sqf [hunk 1]
```diff
@@ -526,11 +555,30 @@ _HLC416C = ["hlc_rifle_416C",_30rnd556AR15];
 _HLC416N = ["hlc_rifle_416N",_30rnd556AR15];
 _HLC416NGL = ["hlc_rifle_416N_gl",_30rnd556AR15,_40mm];
 
-//========== HLC M16 PACK =======================================================
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: Why was this deleted and the weapons moved to `AR PACK`?

File: loadouts/content/broma-weapon-list.sqf [hunk 2]
```diff
@@ -153,18 +150,26 @@ _BCMBLACKJACK = ["hlc_rifle_bcmblackjack",_29rnd300BLK];
 _BUSHMASTER  = ["hlc_rifle_Bushmaster300",_29rnd300BLK];
 _HONEYBADGER = ["hlc_rifle_honeybadger",_29rnd300BLK];
 
+_COLTRO733     = ["hlc_wp_mod733",_30rnd556AR15];
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: `hlc_wp_mod733` M16 PACK

File: loadouts/content/broma-weapon-list.sqf [hunk 3]
```diff
@@ -153,18 +150,26 @@ _BCMBLACKJACK = ["hlc_rifle_bcmblackjack",_29rnd300BLK];
...
 _COLTCARBINE   = ["hlc_rifle_Colt727",_30rnd556AR15];
 _COLTCARBINEGL = ["hlc_rifle_Colt727_GL",_30rnd556AR15,_40mm];
 _COLTCOMMANDO  = ["hlc_rifle_CQBR",_30rnd556AR15];
 _COLTM4A1CH    = ["hlc_rifle_M4a1carryhandle",_30rnd556AR15];
 _COLTM4A1      = ["hlc_rifle_M4",_30rnd556AR15];
-_COLTM4A1GL      = ["hlc_rifle_m4m203",_30rnd556AR15,_40mm];
+_COLTM4A1GL    = ["hlc_rifle_m4m203",_30rnd556AR15,_40mm];
+
+_XM177E2       = ["hlc_wp_XM177E2",_30rnd556AR15];
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: `hlc_wp_mod733` M16 PACK

File: loadouts/content/broma-weapon-list.sqf [hunk 4]
```diff
@@ -153,18 +150,26 @@ _BCMBLACKJACK = ["hlc_rifle_bcmblackjack",_29rnd300BLK];
...
 
 _MK18MOD0 = ["hlc_rifle_mk18mod0",_30rnd556AR15];
 
 _SAMRTAN = ["hlc_rifle_SAMR",_30rnd556AR15SPR];
 _SAMROD  = ["hlc_rifle_samr2",_30rnd556AR15SPR];
 
+_HLCM16A1 = ["hlc_wp_m16a1","hlc_30rnd_556x45_EPR"];
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: M16 PACK: `hlc_wp_m16a1`, `hlc_rifle_A1m203`, `hlc_wp_m16a2`, `hlc_rifle_m203`

File: loadouts/content/broma-weapon-list.sqf [hunk 5]
```diff
@@ -356,28 +361,52 @@ _HLCMP5K = ["hlc_smg_mp5k_PDW",_30rnd9mmMP5];
 _ACRCOMPACTB = ["hlc_rifle_ACR_SBR_black",_30rnd556AR15];
 _ACRCOMPACTT = ["hlc_rifle_ACR_SBR_tan",_30rnd556AR15];
 _ACRCOMPACTG = ["hlc_rifle_ACR_SBR_green",_30rnd556AR15];
+_ACRCOMPACTBGL = ["hlc_rifle_ACR_GL_SBR_black",_30rnd556AR15,_40mm];
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: `_ACRCOMPACTGLB` instead, same change to the other's. E.g. `_ACRCOMPACTTGL` to `_ACRCOMPACTGLT`, `_ACR68MIDTGL` to `_ACR68MIDGLT`, etc.

File: loadouts/content/broma-weapon-list.sqf [hunk 6]
```diff
@@ -356,28 +361,52 @@ _HLCMP5K = ["hlc_smg_mp5k_PDW",_30rnd9mmMP5];
...
 _ACRSHORTG   = ["hlc_rifle_ACR_Carb_green",_30rnd556AR15];
 _ACRSHORTT   = ["hlc_rifle_ACR_Carb_tan",_30rnd556AR15];
+_ACRSHORTBGL = ["hlc_rifle_ACR_GL_Carb_black",_30rnd556AR15,_40mm];
+_ACRSHORTGGL = ["hlc_rifle_ACR_GL_Carb_green",_30rnd556AR15,_40mm];
+_ACRSHORTTGL = ["hlc_rifle_ACR_GL_Carb_tan",_30rnd556AR15,_40mm];
 
 _ACR68COMPACTB = ["hlc_rifle_ACR68_SBR_black",_30rnd68ACR];
 _ACR68COMPACTG = ["hlc_rifle_ACR68_SBR_green",_30rnd68ACR];
 _ACR68COMPACTT = ["hlc_rifle_ACR68_SBR_tan",_30rnd68ACR];
+_ACR68COMPACTBGL = ["hlc_rifle_ACR68_GL_SBR_black,_30rnd68ACR,_40mm];
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: Missing `"`, same issue with `_ACR68COMPACTGGL` and `_ACR68COMPACTTGL`.

File: loadouts/content/broma-weapon-list.sqf [hunk 7]
```diff
@@ -356,28 +361,52 @@ _HLCMP5K = ["hlc_smg_mp5k_PDW",_30rnd9mmMP5];
...
+_ACR68COMPACTTGL = ["hlc_rifle_ACR68_GL_SBR_tan,_30rnd68ACR,_40mm];
+
 _ACR68LONGB    = ["hlc_rifle_ACR68_full_black",_30rnd68ACR];
 _ACR68LONGG    = ["hlc_rifle_ACR68_full_green",_30rnd68ACR];
 _ACR68LONGT    = ["hlc_rifle_ACR68_full_tan",_30rnd68ACR];
+
 _ACR68MIDB     = ["hlc_rifle_ACR68_mid_black",_30rnd68ACR];
 _ACR68MIDG     = ["hlc_rifle_ACR68_mid_green",_30rnd68ACR];
 _ACR68MIDT     = ["hlc_rifle_ACR68_mid_tan",_30rnd68ACR];
+_ACR68MIDBGL   = ["hlc_rifle_ACR68_GL_mid_black,_30rnd68ACR,_40mm];
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: Missing `"`, same issue with `_ACR68MIDGGL` and `_ACR68MIDTGL`.

File: loadouts/content/broma-weapon-list.sqf (hunk 1 above)
Comment (HeroesMaker) [reply loadouts/content/broma-weapon-list.sqf]: oh, they were part of the HLC AR pack, so i decided t oassamble it all together so if someone is looking for their classnames he can found them here ( was there really an M16 pack or was it just the AR pack that got updated ? )

File: loadouts/content/broma-weapon-list.sqf [hunk 8]
```diff
@@ -526,11 +555,30 @@ _HLC416C = ["hlc_rifle_416C",_30rnd556AR15];
...
+_C96PRUSSHORT      = ["hlc_pistol_C96_Prussian_Rework",_10rnd9mmMAUSER];
+_C96PRUSSTOCK      = ["hlc_pistol_C96_Prussian_stock",_10rnd9mmMAUSER];
+_C96PRUSSHORTSTOCK = ["hlc_pistol_C96_Prussian_Rework_stock",_10rnd9mmMAUSER];
+
+_C96M1921          = ["hlc_pistol_C96_Bolo",_10rnd763MAUSER];
+_C96M1921WORN      = ["hlc_pistol_C96_Bolo_Worn",_10rnd763MAUSER];
+_C96M1921STOCK     = ["hlc_pistol_C96_Bolo_stock",_10rnd763MAUSER];
+_C96M1921WORNSTOCK = ["hlc_pistol_C96_Bolo_Worn_stock",_10rnd763MAUSER];
 
+_20rnd763MAUSER
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: Delete

File: loadouts/content/broma-weapon-list.sqf (hunk 1 above)
Comment (HeroesMaker) [reply loadouts/content/broma-weapon-list.sqf]: just checked, they are all part of the AR-15 pack

**Comment by HeroesMaker** [comment]
i guess that should be it for the content list, i check NIArms changelog and all the weapon you listed as M16 PACK were added in the AR-15 pack ver. 2.02

File: loadouts/content/broma-weapon-list.sqf (hunk 1 above)
Comment (Coryf88) [reply loadouts/content/broma-weapon-list.sqf]: Ah, I think I understand what happened. I assume Royal probably added this section based on `@NIArms All in One` PBO's which has `hlc_wp_M16A2.pbo`. However, there isn't actually an M16 pack and that PBO and `hlc_wp_AR15.pbo` is apart of `@NIArms AR15 Rifles`.
